### DISCUSSION TITLE
How to change the background color of the chat windows to dark (or if there is a dark mode?)

### DISCUSSION THREAD

**Discussion by kepler-cpu** [chatscope/chat-ui-kit-react#102] [Q&A]
Hi I am new to react.

the current chat window background is white.  I would like to use dark mode.
Is there a such theme that I can apply?  or how do I change the background color and message box and text color?

thank you
k.

**Comment by supersnager** [comment]
@kepler-cpu There is no ready-made dark mode available. However, you can change scss variables to change colors.
These issues can be helpful:
https://github.com/chatscope/chat-ui-kit-styles/issues/5
https://github.com/chatscope/chat-ui-kit-react/issues/79
